server/seed_reviews.js

```diff
@@ -0,0 +1,53 @@
+const faker = require('faker');
+const { Review } = require('./db.js');
+
+const reviewText = () => {
+  let reviewParagraphs;
+  const reviewLength = Math.ceil(Math.random() * 4);
+  for (let paragraphs = 0; paragraphs < reviewLength; paragraphs++) {
+    reviewParagraphs = reviewParagraphs + '\n' + faker.lorem.paragraph();
```

Comment:
I believe `faker.lorem.paragraph()` takes in an argument for how many paragraphs to generate.